client/state/account-recovery/settings/actions.js

```diff
@@ -287,7 +282,7 @@ export const validateAccountRecoveryPhone = ( code ) => ( dispatch ) => {
 	return wpcom
 		.undocumented()
 		.me()
-		.validateAccountRecoveryPhone( replace( code, /\s/g, '' ) )
+		.validateAccountRecoveryPhone( code?.replace( /\s/g, '' ) )
```

Comment:
`code` will always be a string here: it's coming from an `<input>` value.

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/52234/commits/55c6602dd7f5b9fb2da29ba025322ac84884fec0 and fixed the test that was failing because of that - it was incorrectly calling the thunk without an argument.